manifests/agent.pp

```diff
@@ -173,7 +173,9 @@
   }
 
   # See https://docs.sensu.io/sensu-go/latest/installation/upgrade/
-  if $facts['service_provider'] == 'systemd' {
+  # Only necessary for Puppet < 6.1.0,
```

Comment:
We only support the latest versions of puppet 5 and 6. Given this is an old version, I don't think we should support it. We should document this though.

Reply:
This is necessary to ensure we execute `systemctl daemon-reload` on Puppet 5. Only Puppet >= 6.1.0 has this fix. The latest Puppet 5 does NOT execute `systemctl daemon-reload`. The condition I added ensures we are not executing this for latest Puppet 6 versions where it's unnecessary.